webpack.config.js

```diff
@@ -55,4 +55,7 @@ module.exports = {
 			],
 		}),
 	],
+	externals: {
+		'@wordpress/plugins': 'wp.plugins',
+	}
```

Comment:
This isn't required as it's handled by the `@wordpress/dependency-extraction-webpack-plugin` in wp-scripts.